package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.modeler",
-  "version": "2.3.0",
+  "version": "2.4.0-prerelease-1",
```

Comment:
nice! I have to finish adding the support for this in the scripts :D 

AutoRest extension build scripts will of course put in the third digit from the git commit #.

However, We're going use the suffix text for an NPM tag so we should probably make this `2.4.0-error-response`  otherwise the tag will be `prerelease-1` (which isn't descriptive)